examples/background-worker/chart.ts

```diff
@@ -0,0 +1,73 @@
+import { Construct } from "constructs";
+import { Chart, ChartProps } from "cdk8s";
+import { BackgroundWorker, ResqueWeb, Secret } from "../../lib";
+import { KubeNamespace, Quantity } from "../../imports/k8s";
+
+export class BackgroundWorkerChart extends Chart {
+  constructor(
+    scope: Construct,
+    id: string,
+    { namespace = "example-background-worker", ...props }: ChartProps = {}
+  ) {
+    super(scope, id, { namespace, ...props });
+
+    const release = process.env.RELEASE || "v1.0";
+    const redisUrl = "redis.cache.svc.cluster.local:6379:1";
+
+    new KubeNamespace(this, "namespace", { metadata: { name: namespace } });
+
+    const dockerHubCredentials = new Secret(this, "docker-hub-cred", {
+      type: "kubernetes.io/dockerconfigjson",
+      data: {
+        ".dockerconfigjson": JSON.stringify({
+          auths: {
+            "https://index.docker.io/v1/": {
+              auth: "c29tZXVzZXI6c2VjcmV0MTIz",
```

Comment:
Should this be an env var?

Reply:
This is just an example, not real credentials. I didn't make the examples to be deployable, they're more like "this is how you could use it". Perhaps they should be more complete? : thinking:

Reply:
Yeah even just `auth: process.env.SUPER_SECURE_SECRET` would suffice I think

Reply:
@ShaunParsons updated in 3a5a23575c636cecb1783684cc8c9bbb9aa9a18c